src/openforms/forms/api/viewsets.py

```diff
@@ -757,6 +761,19 @@ def post(self, request, *args, **kwargs):
         serializer = self.serializer_class(data=request.data)
         serializer.is_valid(raise_exception=True)
 
-        import_form(serializer.validated_data["file"])
+        form_instance = import_form(serializer.validated_data["file"])
+        assert form_instance
+
+        response_serializer = FormImportResponseSerializer(instance=form_instance)
 
-        return response.Response(status=status.HTTP_204_NO_CONTENT)
+        return response.Response(
+            status=status.HTTP_201_CREATED,
+            data=response_serializer.data,
+            headers={
+                "Location": request.build_absolute_uri(
+                    reverse(
+                        "api:form-detail", kwargs={"uuid_or_slug": form_instance.uuid}
+                    )
+                )
```

Comment:
actually I remember now that we have a helper function for this: https://github.com/open-formulieren/open-forms/blob/644413b91d391691cb0c5368aa9d4fe062664312/src/openforms/utils/urls.py#L66